### PR TITLE
feat(code-connect): connect additional components

### PR DESCRIPTION
**Pull request by alisonjoseph**
#### Changelog

**New**
- Update to latest code connect version
- Connect following components
  - ComposedModal
  - DatePicker
  - Dropdown
  - FileUploader
  - Form
  - Notification
  - Popover
  - ProgressIndicator
  - Slider
  - TextArea
  - TextInput
  - TimePicker
  - Toggletip
  - Tooltip
  - Menu, MenuItem

### CODE REVIEW

File: packages/react/code-connect/TimePicker/TimePicker.figma.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      }),
+      // labelText: figma.textContent('Choose a time'),
+      // placeholder: figma.textContent('hh:mm'),
+      // missing from Figma
+      // value,
+      // warning,
+      // warningText,
+      // invalid,
+      // invalidText,
+      // hideLabel
```
Comment (ariellalgilmore) [inline packages/react/code-connect/TimePicker/TimePicker.figma.tsx]: thinking we might want some of these?

File: packages/react/code-connect/ComposedModal/ComposedModal.figma.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        true: figma.string('Label text'),
+      }),
+      size: figma.enum('Size', {
+        Large: 'lg',
+        Medium: 'md',
+        Small: 'sm',
+        'Extra small': 'xs',
+      }),
+      progress: figma.boolean('Progress', {
+        // true: <ProgressIndicator />,
```
Comment (ariellalgilmore) [inline packages/react/code-connect/ComposedModal/ComposedModal.figma.tsx]:
```suggestion
```

File: packages/react/code-connect/DatePicker/FluidDatePicker.figma.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      //   Focus: 'focus',
+      //   Error: 'error',
+      //   Disabled: 'disabled',
+      //   Skeleton: 'skeleton',
+      //   'Read-only': 'read-only',
+      //   Warning: 'warning',
+      // }),
+      // open: figma.boolean('Open'),
+      // datefilled: figma.boolean('Date filled'),
+      // tooltip: figma.boolean('Tooltip'),
```
Comment (ariellalgilmore) [inline packages/react/code-connect/DatePicker/FluidDatePicker.figma.tsx]: maybe some of these can be used too?

**Review by ariellalgilmore — COMMENTED** [review]
loooking good! just a few comments i wasn't sure about

**Comment by alisonjoseph** [comment]
@ariellalgilmore thanks for reviewing! I just removed TimePicker from this PR, its not finished yet 😅

**Review by ariellalgilmore — APPROVED** [review]
Awesome LGTM!

**Review by guidari — APPROVED** [review]

**Review by riddhybansal — APPROVED** [review]

**Pull request merged**